nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/authorization/ProcessGroupAuthorizable.java

```diff
@@ -32,13 +33,28 @@ public interface ProcessGroupAuthorizable extends AuthorizableHolder {
      */
     ProcessGroup getProcessGroup();
 
+    /**
+     * Returns the Parameter Context Authorizable. May be null if the underlying Process Group is not
+     * bound to a Parameter Context.
+     *
+     * @return the Parameter Context authorizable
+     */
+    Authorizable getParameterContextAuthorizable();
```

Comment:
If this method can return `null`, would it be better to return it in an `Optional` wrapper?